package.json

```diff
@@ -37,9 +38,10 @@
     "node": ">=0.4.x"
   },
   "dependencies": {
-    "underscore": "1.8.3",
     "mapnik-reference": "~6.0.2",
-    "optimist": "~0.6.0"
+    "optimist": "~0.6.0",
+    "torque-reference": "git+https://github.com/CartoDB/torque-reference.git",
```

Comment:
Ideally, we should publish it to NPM, or, at least, use a tag to keep more under control what version we are targeting.